docs/content/standards/kiosk-apps/nft-rental.mdx

```diff
@@ -0,0 +1,430 @@
+
+```move
+struct Rentables has drop {}
+```
+
+The `Rented` struct represents a rented item. The only field the struct includes is the ID of the object. It is used as the DF key in the borrower's Bag <key, value> entry when an item is actively being borrowed by someone.
+
+```move
+struct Rented has store, copy, drop { 
+    id: ID 
+}
+```
+
+The `Listed` struct represents a listed item. The only field the struct includes is the ID of the object. It is used as the DF key in the renter's Bag <key, value> entry once an item is listed for renting. 
+
+```move
+struct Listed has store, copy, drop { 
+    id: ID 
+}
+```
+
+The `Promise` struct is created for borrowing by value. The `Promise` operates as the [hot potato](../../concepts/sui-move-concepts/patterns/hot-potato.mdx) that can only be resolved by returning the item back to the extension's `Bag`.
+The Promise field lacks the `store` ability as it shouldn't be wrapped inside other objects.
+It also lacks the `drop` ability since it can only be consumed by the `return_val` function.
+
+```move
+struct Promise {
+  item_id: Rented,
+  duration: u64,
+  start_date: u64,
+  price_per_day: u64,
+  renter: address,
+  borrower_kiosk: ID
+}
+```
+
+The `Rentable` struct is as a wrapper object that holds an asset that is being rented. Contains information relevant to the rental period, cost, and renter.
+This struct requires the `store` ability since it stores a value `T` that will definitely also have `store`.
+
+```move
+struct Rentable<T: key + store> has store {
+      object: T,
+      duration: u64, // total amount of time offered for renting in days
+      start_date: Option<u64>, // initially undefined, is updated once someone rents it
+      price_per_day: u64,
+      kiosk_id: ID // the kiosk id that the object was taken from
+}
+```
+
+The `RentalPolicy` struct is a shared object that every creator mints. Defines the royalties the creator receives from each rent invocation.
+
+```move
+struct RentalPolicy<phantom T> has key, store {
+	id: UID,
+	balance: Balance<SUI>,
+	/// Note: Move does not support float numbers.
+	/// If you need to represent a float, you need to determine the desired precision and use a larger integer representation.
+	/// For example, percentages can be represented using basis points:
+	/// 10000 basis points represents 100% and 100 basis points represents 1%.
+	amount_bp: u64
+}
+```
+
+The `ProtectedTP` object is a shared object that every creator, that would like to enable renting, mints, regardless of their intent to enforce royalties. The object provides authorized access to an empty `TransferPolicy`.
+A protected empty transfer policy is required to facilitate the rental process so that the extension can transfer the asset without any additional rules to be resolved (like lock rule, loyalty rule, etc).
```

Comment:
```suggestion
A protected empty transfer policy is required to facilitate the rental process so that the extension can transfer the asset without any additional rules to be resolved (like lock rule, loyalty rule, and so on).
```
Don't use latin abbr because it's not everyone's base.